ui/src/config/section/compute.js

```diff
@@ -181,6 +181,9 @@ export default {
           docHelp: 'adminguide/virtual_machines.html#virtual-machine-snapshots',
           dataView: true,
           popup: true,
+          show: (record) => {
+            return ((['Running'].includes(record.state) && record.hypervisor !== 'LXC'))
```

Comment:
@utchoang It can still be visible when the VM is stopped and not KVM or LXC

https://github.com/apache/cloudstack/blob/4.15/ui/legacy/scripts/instances.js#L4210